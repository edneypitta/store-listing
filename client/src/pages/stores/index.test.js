import React from 'react'
import { render } from 'enzyme'
import StoresPage from './index'
import CategoriesContext from '../../categories-context'

describe('StoresPage', () => {
  it('renders without crashing', () => {
    const category = {
      name: 'gifts',
      stores: [{ tags: [] }]
    }
    const component = render(
      storesPageComponent(category))

    expect(component).toMatchSnapshot()
  })
})

function storesPageComponent(category) {
  return (
    <CategoriesContext.Provider value={[category]}>
      <StoresPage location={{ search: 'category=gifts' }} />
    </CategoriesContext.Provider>)
}

