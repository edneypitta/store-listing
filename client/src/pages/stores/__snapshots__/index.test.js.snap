// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoresPage renders without crashing 1`] = `
<div
  class="stores"
>
  <div
    class="store"
  >
    <h2 />
    <p />
    <span>
      Unknown opening times
    </span>
    <div
      class="tags"
    />
  </div>
</div>
`;
